sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md

```diff
@@ -0,0 +1,398 @@
+
+```C# Snippet:DocumentIntelligenceSampleFormulaExtraction
+Uri uriSource = new Uri("<uriSource>");
+
+var content = new AnalyzeDocumentContent()
+{
+    UrlSource = uriSource
+};
+
+List<DocumentAnalysisFeature> features = new List<DocumentAnalysisFeature>();
+features.Add(DocumentAnalysisFeature.Formulas);
+
+var operation = await client.AnalyzeDocumentAsync(WaitUntil.Completed, "prebuilt-layout", content, features: features);
+AnalyzeResult result = operation.Value;
+
+foreach (DocumentPage page in result.Pages)
+{
+    Console.WriteLine($"----Formulas detected from page #{page.PageNumber}----");
+
+    Console.WriteLine($"Detected {page.Formulas.Count} formulas.");
+    for (int i = 0; i < page.Formulas.Count; i++)
+    {
+        DocumentFormula formula = page.Formulas[i];
+        Console.WriteLine($"- Formula #{i}: {formula.Value}");
+        Console.WriteLine($"  Kind: {formula.Kind}");
+        Console.WriteLine($"  Confidence: {formula.Confidence}");
+        Console.WriteLine($"  bounding polygon (points ordered clockwise):");
+        for (int j = 0; j < formula.Polygon.Count; j += 2)
+        {
+            Console.WriteLine($"      Point {j / 2} => X: {formula.Polygon[j]}, Y: {formula.Polygon[j + 1]}");
+        }
+    }
+}
+```
+
+## Font property extraction
+To extract font information from a given file at a URI with the add-on font styling capability, use the `AnalyzeDocumentAsync` method and specify `DocumentAnalysisFeature.FontStyling` as the analysis features. The returned value is an `AnalyzeResult` object containing data about the submitted document.
+
+```C# Snippet:DocumentIntelligenceSampleFontStyling
+Uri uriSource = new Uri("<uriSource>");
+
+var content = new AnalyzeDocumentContent()
+{
+    UrlSource = uriSource
+};
+
+List<DocumentAnalysisFeature> features = new List<DocumentAnalysisFeature>();
+features.Add(DocumentAnalysisFeature.StyleFont);
+
+var operation = await client.AnalyzeDocumentAsync(WaitUntil.Completed, "prebuilt-layout", content, features: features);
+AnalyzeResult result = operation.Value;
+
+// DocumentStyle has the following font related attributes:
+var similarFontFamilies = new Dictionary<string, List<DocumentStyle>>(); // e.g., 'Arial, sans-serif
+var fontStyles = new Dictionary<FontStyle, List<DocumentStyle>>(); // e.g, 'italic'
+var fontWeights = new Dictionary<FontWeight, List<DocumentStyle>>(); // e.g., 'bold'
+var fontColors = new Dictionary<string, List<DocumentStyle>>(); // in '#rrggbb' hexadecimal format
+var fontBackgroundColors = new Dictionary<string, List<DocumentStyle>>(); // in '#rrggbb' hexadecimal format
+
+if (result.Styles.Any(s => s.IsHandwritten != null && s.IsHandwritten.Value == true))
+{
+    Console.WriteLine("Document contains handwritten content");
+}
+else
+{
+    Console.WriteLine("Document does not contain handwritten content");
+}

```

Comment:
How is this helpful to customers? I guess they would want to extract what exactly is the handwritten content instead.

Reply:
Print the handwritten content instead.